internal/controller/services/monitoring/monitoring_controller.go

```diff
@@ -110,13 +115,16 @@ func (h *serviceHandler) NewReconciler(ctx context.Context, mgr ctrl.Manager) er
 		WithAction(initialize).
 		WithAction(updatePrometheusConfigMap).
 		WithAction(createMonitoringStack).
+		WithAction(deployTempo).
 		WithAction(template.NewAction(
 			template.WithDataFn(getTemplateData),
 		)).
 		WithAction(deploy.NewAction(
 			deploy.WithCache(),
 		)).
-		WithAction(gc.NewAction()).
+		WithAction(gc.NewAction(
+			gc.InNamespaceFn(monitoringNamespace),
```

Comment:
think we do not need explicitly set it here

Reply:
that came from the rebase